content/docs/1.6.0/deploy/important-notes/index.md

```diff
@@ -106,3 +106,26 @@ Starting from `v1.5.0`, the following deprecated custom resource fields will be
 - BackingImageManager.status.backingImageFileMap.directory
 - BackingImageManager.status.backingImageFileMap.downloadProgress
 - BackingImageManager.status.backingImageFileMap.url
+
+### Engine Upgrade Enforcement
+
+Starting from `v1.6.0`, Longhorn will implement mandatory version upgrades for the engines. The minimum required supporting version will be outlined in the [release note](https://github.com/longhorn/longhorn/releases/tag/v{{< current-version >}}).
```

Comment:
Beginning with version `v1.6.0`, Longhorn will implement mandatory engine upgrades. See the [release note](https://github.com/longhorn/longhorn/releases/tag/v{{< current-version >}}) for information about the minimum supported engine image version.